src/Components/Forms/FormCreateUpdate.js

```diff
@@ -0,0 +1,316 @@
+import React from "react";
+import ButtonsCreate from "./ButtonsCreate";
+import ButtonsUpdate from "./ButtonsUpdate";
+import Input from "./Input";
+import ListItems from "./ListItems";
+import styles from "./FormCreateUpdate.module.css";
+import { GlobalContext } from "../../Context/GlobalMode";
+import { ReactComponent as ArrowLeft } from './../../Assets/icon-arrow-left.svg';
+
+const FormCreateUpdate = ({
+    typeForm,
+    address1,
+    address2,
+    city1,
+    city2,
+    code1,
+    code2,
+    country1,
+    country2,
+    name,
+    email,
+    invoiceDate,
+    project,
+    items,
+    setItems,
+    closeModal,
+    payment,
+    setPayment,
+    saveInvoice,
+    id,
+}) => {
+    const [error, setError] = React.useState(false);
+    const [error2, setError2] = React.useState(false);
+    const global = React.useContext(GlobalContext);
```

Comment:
Bolje je izvuci samo sta treba tj. destructuring
```suggestion
    const {mode} = React.useContext(GlobalContext);
```